src/main/java/com/ea/restaurant/entities/BaseEntity.java

```diff
@@ -0,0 +1,24 @@
+package com.ea.restaurant.entities;
+
+import com.ea.restaurant.constants.Status;
+import java.time.Instant;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@Getter
+@Setter
+public abstract class BaseEntity {
+
+  @EqualsAndHashCode.Include private long id;
```

Comment:
could you specify the id in `EqualsAndHashCode` on the class definition? this will help to keep the class as cleaner as possible

Reply:
done.